vcd/resource_vcd_vm_internal_disk_test.go

```diff
@@ -0,0 +1,479 @@
+// +build vapp vm ALL functional
+
+package vcd
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+	"regexp"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+)
+
+func TestAccVcdVmInternalDisk(t *testing.T) {
+	// This test requires access to the vCD before filling templates
+	// Thus it won't run in the short test
+	if vcdShortTest {
+		t.Skip(acceptanceTestsSkipped)
+		return
+	}
+	// In general VM internal disks works with Org users, but due we need create VDC with disabled fast provisioning value, we have to be sys admins
```

Comment:
```suggestion
	// In general VM internal disks works with Org users, but since we need to create VDC with disabled fast provisioning value, we have to be sys admins
```

Reply:
updated